bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,98 @@
+                  "https://cdn.discordapp.com/attachments/859123972884922418/883472808699629578/hangman2.png",
+                  "https://cdn.discordapp.com/attachments/859123972884922418/883472862441267230/hangman3.png",
+                  "https://cdn.discordapp.com/attachments/859123972884922418/883472950991396864/hangman4.png",
+                  "https://cdn.discordapp.com/attachments/859123972884922418/883472999431430204/hangman5.png",
+                  "https://cdn.discordapp.com/attachments/859123972884922418/883473051277226015/hangman6.png"]
+
+        # a dictionary mapping the images of the "hung man" to the number of tries it corresponds to
+        mapping_of_images = {tries_key: image_name for image_name, tries_key in zip(images, range(6, -1, -1))}
+
+        word = choice(filtered_words)
+        user_guess = "_" * len(word)
+        tries = 6
+        hangman_embed = Embed(title="Hangman", color=Color.blurple())
+        hangman_embed.set_image(url=mapping_of_images[tries])
+        hangman_embed.add_field(name=f"The word is `{user_guess}`", value="Guess the word by sending a "
+                                                                          "message with the letter!", inline=False)
+        original_message = await ctx.send(embed=hangman_embed)
+
+        while user_guess != word:
+            try:
+                message = await self.bot.wait_for(event="message", timeout=30.0,
+                                                  check=lambda msg: msg.author != self.bot)
+
+                if len(message.content) > 1:
+                    continue
+
+                elif message.content in word:
+                    positions = [idx for idx, letter in enumerate(word) if letter == message.content]
+                    user_guess = "".join([message.content if index in positions else dash
+                                          for index, dash in enumerate(user_guess)])
+
+                elif tries - 1 <= 0:
+                    losing_embed = Embed(title="You lost.", description=f"The word was `{word}`.", color=Color.red())
+                    await original_message.edit(embed=losing_embed)
+                    return
+
+                else:
+                    tries -= 1
+
+                hangman_embed.clear_fields()
+                hangman_embed.set_image(url=mapping_of_images[tries])
+                hangman_embed.add_field(name=f"The word is `{user_guess}`", value="Guess the word by sending a "
+                                                                                  "message with the letter!",
+                                        inline=False)
+                await original_message.edit(embed=hangman_embed)
+            except TimeoutError:
+                return
+        else:
+            win_embed = Embed(title="You won!", color=Color.green())
+            await ctx.send(embed=win_embed)
+
+
+def setup(bot: Bot) -> None:
+    """Setting up the Hangman cog."""
```

Comment:
I believe the convention for these setup functions' docstrings is `"""Load the {name} cog."""`. This is definitely not necessary, but changing it for the sake of consistency would be nice.